aips/aip-x.md

```diff
@@ -0,0 +1,707 @@
+| fields.domain         | string          | ✓         | `dnsauthority` that is requesting the signing                                                                                         |
+| fields.address        | string          | ✓         | Blockchain address performing the signing.                                                                                            |
+| fields.uri            | string          | ✓         | URI referring to the resource that is the subject of the signing i.e. the subject of the claim.                                       |
+| fields.version        | string          | ✓         | Current version of the message.                                                                                                       |
+| fields.statement      | string          |           | Human-readable ASCII assertion that the user will sign. It MUST NOT contain `\n`.                                                     |
+| fields.chainId        | string          | ✓         | Identifier for the network locating the address listed separately above.                                                              |
+| fields.nonce          | string          |           | Randomized token to prevent signature replay attacks.                                                                                 |
+| fields.issuedAt       | ISO 8601 string |           | `date-time` that indicates the issuance time.                                                                                         |
+| fields.expirationTime | ISO 8601 string |           | `date-time` that indicates when the signed authentication message is no longer valid.                                                 |
+| fields.notBefore      | ISO 8601 string |           | `date-time` that indicates when the signed authentication message starts being valid.                                                 |
+| fields.requestId      | string          |           | System-specific identifier used to uniquely refer to the authentication request.                                                      |
+| fields.resources      | string[]        |           | List of information or references to information the user wishes to have resolved as part of the authentication by the relying party. |
+| plainText             | string          | ✓         | The constructed message using the standardized message format.                                                                        |
+| signature             | bytes           | ✓         | The signature of the message. This must conform to the domain separated signing algorithm.                                            |
+| type                  | string          | ✓         | The public key scheme used to sign the message (e.g. 'ed25519', 'single_key', 'multi_key', 'multi_ed25519', etc.)                     |
+
+### AptosSignInOutput
```

Comment:
I don't like this generic "output" name. Shouldn't this be something more like a `AptosSignInSignature` or `SiwaSignature`?